src/murmur/Messages.cpp

```diff
@@ -77,6 +77,22 @@
 		sendMessage(uSource, mppd); \
 	}
 
+#ifdef USE_GRPC
+#include <boost/fiber/all.hpp>
+#define SERVER_USER_ALIVE(user, session) \
+do { \
+		while(!qrwlVoiceThread.tryLockForRead()) { \
+			::boost::this_fiber::yield(); \
+		} \
+		if (!qhUsers.contains(session)) { \
+			qrwlVoiceThread.unlock(); \
+			return; \
+		} \
+		qrwlVoiceThread.unlock(); \
+	} while (false)
```

Comment:
Indentation seems to mis-match

Reply:
Hadn't set up my .vimrc correctly for this project yet -- some of the spacing is still off as I prefer 4 space tabs. As I said before, it needs cleanup.

Reply:
That's why I simply use tabs instead of spaces :P

I only added this comment because it caught my eyes while investigating the `while (false)` case ^^